tox.ini

```diff
@@ -9,14 +9,14 @@ commands =
 
 [testenv:black]
 deps =
-    black == 22.12.0
+    black == 3.7.0
```

Comment:
```suggestion
    black == 23.7.0
```

Reply:
Ahead of you this time :)

Reply:
I like the idea of pinning a version of black from 2003 :)